src/gpu/amd/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+#===============================================================================
+# Copyright 2020-2021 Intel Corporation
+# Copyright 2020 Codeplay Software Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+file(GLOB_RECURSE SOURCES
+    ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
+
+set(OBJ_LIB ${DNNL_LIBRARY_NAME}_sycl_miopen)
```

Comment:
(nit) For NVIDIA it's `set(OBJ_LIB ${DNNL_LIBRARY_NAME}_sycl_nvidia)` so for AMD is should probably be `set(OBJ_LIB ${DNNL_LIBRARY_NAME}_sycl_amd)`

Reply:
Addressed